### DISCUSSION TITLE
Hidden column not remaining disabled

### DISCUSSION THREAD

**Discussion by Dragon-32** [X2CommunityCore/xcom2-launcher#288] [General]
**AML Version:** 1.5.0-beta.2

**Description**
Options > Show hidden mods is *disabled*
Hidden column in main view is *disabled*

Options > Show hidden mods is *enabled*
Hidden column in main view is *enabled* automatically

The behaviour I expect is that the Hidden column remains disabled regardless of the state of the main window and is disabled or enabled only on user action.

I'll leave it up to those in the know as to whether this is a bug or a conscious design decision.

**Comment by RevZero** [comment]
The reasoning behind this is, that when "show hidden" is disabled, the "Hidden" column serves no purpose and thus is automatically hidden from the list view. For consistency, the column also gets automatically enabled when "show hidden" is enabled.

**Comment by Dragon-32** [comment]
> For consistency, the column also gets automatically enabled when "show hidden" is enabled.

I guess that's the bit I disagree with, but if it's a conscious decision then cool beans.

**Comment by RevZero** [comment]
I will move this to discussions then.
